ansible_collections/arista/cvp/molecule/cv_device_v3/test_cv_device_v3.yaml

```diff
@@ -0,0 +1,95 @@
+---
+- name: Test cv_device_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: no
+  vars:
+    CVP_DEVICES_MOVE_DEVICE:
+      - fqdn: leaf1
+        parentContainerName: "Spine"
```

Comment:
I think you should move this to `Leaf` container just to make things logical

Reply:
fixed